airbase-policy/src/main/resources/checkstyle/airbase-checks.xml

```diff
@@ -63,23 +63,27 @@
     </module>
 
     <module name="RegexpSingleline">
-        <property name="format" value="^([^i]|i[^m]|im[^p]|imp[^o]|impo[^r]|impor[^t]|import[^ ]).*Objects\.requireNonNull" />
+        <property name="format" value="^(?!import ).*Objects\.requireNonNull" />
         <property name="message" value="Objects.requireNonNull should only be used with static imports" />
     </module>
     <module name="RegexpSingleline">
-        <property name="format" value="^([^i]|i[^m]|im[^p]|imp[^o]|impo[^r]|impor[^t]|import[^ ]).*Math\.toIntExact" />
+        <property name="format" value="^(?!import ).*Math\.toIntExact" />
         <property name="message" value="Math.toIntExact should only be used with static imports" />
     </module>
     <module name="RegexpSingleline">
-        <property name="format" value="^([^i]|i[^m]|im[^p]|imp[^o]|impo[^r]|impor[^t]|import[^ ]).*ImmutableMap\.toImmutableMap" />
+        <property name="format" value="^(?!import ).*String\.format" />
```

Comment:
can we use whole package name here?

Reply:
I think no, because usually we have package name static imported.